ckanext/datastore/controller.py

```diff
@@ -28,7 +28,7 @@
 boolean_validator = get_validator('boolean_validator')
 
 DUMP_FORMATS = 'csv', 'tsv', 'json', 'xml'
-PAGINATE_BY = 10000
+PAGINATE_BY = 32000
```

Comment:
it comes from these benchmarks https://github.com/ckan/ckan/pull/3523#issuecomment-296304558 : there's not much benefit to going higher but 32k is a pretty big improvement over 10k, especially for csv